app/controllers/users_controller.rb

```diff
@@ -5,11 +5,21 @@ def show
     conn = Faraday.new(url: 'https://api.github.com') do |faraday|
       faraday.headers['Authorization'] = "token #{current_user.github_token}"
     end
-    github_resp = conn.get('/user/repos')
-    @parsed_resp = JSON.parse(github_resp.body, symbolize_names: true)[0..4]
-    @parsed_resp.map! do |resp|
+    repo_resp = conn.get('/user/repos')
+    @parsed_repo_resp = JSON.parse(repo_resp.body, symbolize_names: true)[0..4]
+    @parsed_repo_resp.map! do |resp|
       Repo.new(resp)
     end
+    followers_resp = conn.get('user/followers')
+    @parsed_follow_resp = JSON.parse(followers_resp.body, symbolize_names: true)
+    @parsed_follow_resp.map! do |follower|
+      Follower.new(follower)
+    end
+    following_resp = conn.get('user/following')
+    @parsed_flwing_resp = JSON.parse(following_resp.body, symbolize_names: true)
```

Comment:
see if we can figure out a way to keep it below 80 characters but make it more clear than "flwing"